### PR TITLE
[Improve#3068] Improve streampark-console module base on [3.5 Concurrent Processing]

### LINKED ISSUE #3068: [Improve] Improve `streampark-console` module base on [3.5 Concurrent Processing]
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-streampark/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

[Improve] Improve `streampark-console` module base on [3.5 Concurrent Processing](https://streampark.apache.org/community/submit_guide/code_style_and_quality_guide#35-concurrent-processing)

### Usage Scenario

_No response_

### Related issues

#2986 

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhengkezhou1**
## What changes were proposed in this pull request

Issue Number: close #3068

## Brief change log
Unified definition of thread pool

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (no)

### CODE REVIEW

**Comment by wolfboys** [comment]
Thanks for your recent positive contributions. Keep up the good work. Let's continue working together to make the project better!

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/config/AsyncExecutorPoolConfig.java [hunk 1]
```diff
@@ -40,4 +40,107 @@ public Executor taskExecutor() {
...
+    ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
+
+    executor.setCorePoolSize(Runtime.getRuntime().availableProcessors() * 5);
+    executor.setMaxPoolSize(Runtime.getRuntime().availableProcessors() * 10);
+    executor.setQueueCapacity(1024);
+    executor.setKeepAliveSeconds(60);
+    executor.setThreadNamePrefix("trigger-savepoint-executor-");
+    executor.setRejectedExecutionHandler(new ThreadPoolExecutor.AbortPolicy());
+    return executor;
+  }
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/config/AsyncExecutorPoolConfig.java:60]: good job

**Comment by zhengkezhou1** [comment]
Next step I'll try to replace the Thread Pool created in the code

**Comment by caicancai** [comment]
> Next step I'll try to replace the Thread Pool created in the code

Looking forward to your next contribution

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**